### DISCUSSION TITLE
Theme how to / migrating a theme from neuron

### DISCUSSION THREAD

**Discussion by kierun** [srid/emanote#116] [Q&A] [answered]
I have been using a CSS theme stored in `~/zettelkasten/static/styles.css`. I migrated from `neuron` to `emanote` and, as expected, the CSS is no longer loaded.

The documentation on how to do a migration, or have a new theme is unclear.

**Comment by srid** [comment, marked as answer]
You can add this to index.yaml, example: https://github.com/srid/www.srid.ca/blob/99f111faaa1bfd442e88d7a257e8ef2f429ff7d4/index.yaml#L4-L7

---

An alternative, but a more general (and global) approach would involve HTML templates and "layer" overriding.

I haven't gotten around to documenting HTML templates yet. In short, look up the relative path of a file under [the default layer](https://github.com/srid/emanote/tree/master/default) and create the same file in your zettelkasten to "override" it. 

Templates have "hook" includes, and the one you need to include stuff in `<head>` is https://github.com/srid/emanote/blob/master/default/templates/hooks/more-head.tpl

So create a `templates/hooks/more-head.tpl` in your Zettelkasten, and write the HTML to include that css file. (You'll have to copy over the default layer's contents if you want to retain the default Nunito font).

**Reply by kierun** [reply]
Thank you very much!
